packages/widgets/src/components/nk-drop-price-text/nk-drop-price-text.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, Host, h, State } from '@stencil/core';
+import state from '../../stores/wallet';
+import { watchBlockNumber } from '@wagmi/core';
+
+@Component({
+  tag: 'nk-drop-price-text',
+  shadow: true,
+})
+export class NKDropPriceText {
+  @State() price: number;
+
+  disconnect: () => void;
+
+  componentWillLoad() {
+    this.disconnect = watchBlockNumber({ listen: true }, async () => {
+      this.price = (await state.diamond.apps.drop.price()).toNumber();
+    });
+  }
+
+  disconnectedCallback() {
+    this.disconnect();
+  }
+
+  render() {
+    return (
+      <Host>
+        {this.price}
```

Comment:
Do we need to convert the price to eth here, it looks like it's in wei on the frontend.

Reply:
Updated per feedback